packages/browser/test/unit/transports/base.test.ts

```diff
@@ -1,133 +0,0 @@
-import { BaseTransport } from '../../../src/transports/base';
-
-const testDsn = 'https://123@sentry.io/42';
-const envelopeEndpoint = 'https://sentry.io/api/42/envelope/?sentry_key=123&sentry_version=7';
-
-// @ts-ignore We're purposely not implementing the methods of the abstract `BaseTransport` class in order to be able to
-// assert on what the class provides and what it leaves to the concrete class to implement
-class SimpleTransport extends BaseTransport {}
-
-// TODO(v7): Re-enable these tests with client reports
```

Comment:
If we delete this we might forget to re add tests. Can we add a note somewhere (maybe in the BrowserClient tests) to add those client report tests back in?